h2o-algos/build.gradle

```diff
@@ -12,9 +12,29 @@ dependencies {
 
   // Jama dependencies
   compile "gov.nist.math:jama:1.0.3"
+
   // netlib-java / MTJ dependencies
-  compile 'com.github.fommil.netlib:all:1.1.2'
-  compile('com.googlecode.matrix-toolkits-java:mtj:1.0.4') { exclude group: 'com.github.fommil.netlib' }
+
+  // Manually define all dependencies in com.github.fommil.netlib:all:1.1.2 to make Databricks maven resolver happy
+  compile('com.github.fommil.netlib:core:1.1.2')
+  compile('net.sourceforge.f2j:arpack_combined_all:0.1')
+  compile('com.github.fommil.netlib:netlib-native_ref-osx-x86_64:1.1')
+  compile('com.github.fommil.netlib:netlib-native_ref-linux-x86_64:1.1')
+  compile('com.github.fommil.netlib:netlib-native_ref-linux-i686:1.1')
+  compile('com.github.fommil.netlib:netlib-native_ref-win-x86_64:1.1')
+  compile('com.github.fommil.netlib:netlib-native_ref-win-i686:1.1')
+  compile('com.github.fommil.netlib:netlib-native_ref-linux-armhf:1.1')
+  compile('com.github.fommil.netlib:netlib-native_system-osx-x86_64:1.1')
+  compile('com.github.fommil.netlib:netlib-native_system-linux-x86_64:1.1')
+  compile('com.github.fommil.netlib:netlib-native_system-linux-i686:1.1')
+  compile('com.github.fommil.netlib:netlib-native_system-linux-armhf:1.1')
+  compile('com.github.fommil.netlib:netlib-native_system-win-x86_64:1.1')
+  compile('com.github.fommil.netlib:netlib-native_system-win-i686:1.1')
+
+  compile('com.googlecode.matrix-toolkits-java:mtj:1.0.4'){
+    exclude group: 'com.github.fommil.netlib', module: 'all'
```

Comment:
We explicitly exclude `all` artifact`. This is semantically equivalent to `exclude group: 'com.github.fommil.netlib'` as there are no additional `netlib` artifacts